setup.py

```diff
@@ -13,7 +13,7 @@ def parse_requirements(filename):
     return [line for line in lineiter if line and not line.startswith("#")]
 
 
-news = os.path.join(os.path.dirname(__file__), 'docs', 'news.rst')
+news = os.path.join(os.path.dirname(__file__), 'docs', 'changelog.rst')
 news = open(news).read()
```

Comment:
better
```python
with open(news) as f:
    news = f.read()
```

Reply:
sure. what if we merged this as is and create an issue to update syntax for setup.py and conf.py in a separate issue? this pr is only for updating the documentation to ensure it's current. we can totally make these other changes but because it's a big PR and these changes aren't in scope for these issues - we could make them in a separate pr. In face - @yihong0618 would you like to open an issue about these changes and then submit a PR?